### PR TITLE
Sweep Test

### PR DESCRIPTION
**Pull request by lealzhan**
Re: #https://github.com/cocos/3d-tasks/issues/15489

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: cocos/physics/cannon/cannon-world.ts [hunk 1]
```diff
@@ -76,6 +76,36 @@ export class CannonWorld implements IPhysicsWorld {
...
     }
 
+    sweepBox (worldRay: geometry.Ray, halfExtent: IVec3Like, orientation: IQuatLike,
+        options: IRaycastOptions, pool: RecyclePool<PhysicsRayResult>, results: PhysicsRayResult[]): boolean {
+        throw new Error('sweepBox not support in cannon.js.');
+    }
+
+    sweepBoxClosest (worldRay: geometry.Ray, halfExtent: IVec3Like, orientation: IQuatLike,
+        options: IRaycastOptions, result: PhysicsRayResult): boolean {
+        throw new Error('sweepBoxClosest not support in cannon.js.');
```
Comment (yiwenxue) [inline cocos/physics/cannon/cannon-world.ts]: should use EngineErrorMap to reduce code size.

File: cocos/physics/cocos/builtin-world.ts [hunk 2]
```diff
@@ -48,6 +48,36 @@ const TriggerEventObject = {
  * not a full physical simulator
  */
 export class BuiltInWorld implements IPhysicsWorld {
+    sweepBox (worldRay: geometry.Ray, halfExtent: IVec3Like, orientation: IQuatLike,
+        options: IRaycastOptions, pool: RecyclePool<PhysicsRayResult>, results: PhysicsRayResult[]): boolean {
+        throw new Error('sweepBox not support in builtin');
```
Comment (yiwenxue) [inline cocos/physics/cocos/builtin-world.ts]: same, use EngineErrorMap

File: cocos/physics/cannon/cannon-world.ts (hunk 1 above)
Comment (lealzhan) [reply cocos/physics/cannon/cannon-world.ts]: ok

File: cocos/physics/cocos/builtin-world.ts (hunk 2 above)
Comment (lealzhan) [reply cocos/physics/cocos/builtin-world.ts]: ok

**Review by yiwenxue — APPROVED** [review]

**Review by pandamicro — APPROVED** [review]

**Pull request merged**